content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -102,14 +102,30 @@ Pods, Controllers and Services are critical elements to managing Windows workloa
 * [azureDisk](/docs/concepts/storage/volumes/#azuredisk)
 * [azureFile](/docs/concepts/storage/volumes/#azurefile)
 * [gcePersistentDisk](/docs/concepts/storage/volumes/#gcepersistentdisk)
+* [awsElasticBlockStore](/docs/concepts/storage/volumes/#awselasticblockstore)
+* [vsphereVolume](/docs/concepts/storage/volumes/#vspherevolume)
+
+##### FlexVolume Plugins
+Code associated with [FlexVolume](/docs/concepts/storage/volumes/#flexVolume) plugins ship as out-of-tree scripts or binaries that need to be deployed directly on the host. FlexVolume plugins handle attaching/detaching of volumes to/from a Kubernetes node and mounting/dismounting a volume to/from individual containers in a pod. Provisioning/De-provisioning of persistent volumes associated with FlexVolume plugins may be handled through an external provisioner that is typically separate from the FlexVolume plugins. The following FlexVolume [plugins](https://github.com/Microsoft/K8s-Storage-Plugins/tree/master/flexvolume/windows), deployed as powershell scripts on the host, support Windows nodes:
+* [SMB](https://github.com/microsoft/K8s-Storage-Plugins/tree/master/flexvolume/windows/plugins/microsoft.com~smb.cmd)
+* [iSCSI](https://github.com/microsoft/K8s-Storage-Plugins/tree/master/flexvolume/windows/plugins/microsoft.com~iscsi.cmd)
+
+##### CSI Plugins
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+Code associated with [CSI](docs/concepts/storage/volumes/#csi) plugins ship as out-of-tree scripts and binaries that are typically packaged in containers and deployed using standard Kubernetes constructs like DaemonSets and StatefulSets. CSI plugins handle a wide range of volume management actions in Kubernetes: Provisioning/De-provisioning/resizing of volumes, attaching/detaching of volumes to/from a Kubernetes node and mounting/dismounting a volume to/from individual containers in a pod, backup/restore of persistent data using snapshots and cloning. CSI plugins typically comprise of node plugins (that run on each node as a DaemonSet) and controller plugins.
+
+CSI node plugins (especially those associated with persistent volumes exposed as either block devices or over a shared file-system) need to perform various privileged operations like scanning of disk devices, mounting of file systems, etc. These operations differ for each host operating system. For Linux worker nodes, containerized CSI node plugins are typically deployed as privileged containers. For Windows worker nodes, privileged operations for containerized CSI node plugins is supported using (csi-proxy)[https://github.com/kubernetes-csi/csi-proxy], a community-managed, stand-alone binary that needs to be pre-installed on each Windows node. Please refer to the deployment guide of the CSI plugin you wish to deploy for further details.
```

Comment:
Check the csi-proxy link. You may need to put the link name in parenthesis on the line with the actual link.